src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -103,15 +104,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     noteEntity.setProperty("categories", note.classifyText());
     noteEntity.setProperty("timestamp", timestamp);
 
+    OutputStream outStream = new ByteArrayOutputStream();
     try {
-      note.writeConvertedDoc();
+      note.writeConvertedDoc(outStream);
     } catch (Docx4JException e) {
       System.out.println(e);
     }
 
     Session session = new Session();
     String objectName = note.getFileName() + ".docx";
-    session.uploadObject(objectName, note.getFilePath());
+    
+    session.uploadObject(objectName, ((ByteArrayOutputStream)(outStream)).toByteArray());
```

Comment:
Just out of curiosity, if two users upload two files with the same name, will the new file overwrite the old one? Or does Cloud Storage give you a unique url for each upload?

Reply:
I believe Cloud Storage gives a unique URL for each object. However, in our case, files shouldn’t have the same names since the file name is based on the unique session id

Reply:
I believe Cloud Storage gives a unique URL for each object. However, in our case, files shouldn’t have the same names since the file name is based on the unique session id